server/src/test/java/org/elasticsearch/cluster/coordination/PersistedStateIT.java

```diff
@@ -33,7 +33,7 @@
 public class PersistedStateIT extends ESIntegTestCase {
 
     private static Settings SETTINGS = Settings.builder()
-        .put(TestZenDiscovery.USE_ZEN2.getKey(), true)
+        .put(TestZenDiscovery.USE_ZEN2.getKey(), getUseZen2())
```

Comment:
this test was meant to specifically test Zen2
so maybe just set it to true here.
Note that this test can probably go away once state recovery is implemented.